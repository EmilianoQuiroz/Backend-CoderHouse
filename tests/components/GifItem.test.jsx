import { render } from '@testing-library/react';
import { GifItem } from '../../src/components/GifItem';
describe('Pruebas en GifItem', () => {

    const title = 'Luffy';
    const url = 'https://lufy.com/lufy.jpg';

    test('Tiene que hacer match con el snapshot', () => {

        const { container } = render( <GifItem title={ title } url={ url } /> );
        expect(container).toMatchSnapshot();
        
    });
});