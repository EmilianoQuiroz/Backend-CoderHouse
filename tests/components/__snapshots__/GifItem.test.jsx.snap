// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en GifItem Tiene que hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Luffy"
      src="https://lufy.com/lufy.jpg"
    />
    <p>
      Luffy
    </p>
  </div>
</div>
`;
